guides/basic-use/folder-layout-and-naming-conventions.md

```diff
@@ -145,3 +145,64 @@ The `app/index.html` file lays the foundation for the Ember application. This is
 In addition to this, the file includes hooks - `{{content-for 'head'}}` and `{{content-for 'body'}}` - that can be used by addons to inject content into the application’s `head` or `body`. 
 
 These hooks need to be left in place for the application to function properly however, they can be safely ignored unless you are directly working with a particular addon.
+
+## Naming conventions
+
+The `Ember Resolver` is the mechanism responsible for looking up code in your application and resolving its dependencies. Ember's convention over configuration philosophy uses naming conventions to support the resolver. While Ember does the work for you when you use the Ember CLI (`ember generate`) to create routes, components or other modules, developers may create files manually.
```

Comment:
I think we can simplify the paragraph and help the reader understand better what the Ember Resolver does for them and what they are to do. What do you think about:

```suggestion
[Ember Resolver](https://github.com/ember-cli/ember-resolver) is responsible for looking up code in your application and resolving its dependencies. The Resolver asks that you follow Ember's naming conventions.

If you use Ember CLI (`ember generate`) to create components, routes, etc., Ember will create files with correct names. In case you may manually create files, let's go over the naming conventions.
```

?